pages/matchs/matchs.module.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  align-items: center;
+  justify-content: space-evenly;
+  margin: 100px auto;
+  width: 35%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-content: stretch;
+}
+
+.nameTeam {
+  background-color: white;
+  color: #333;
+  border-radius: 10px;
+  align-items: center;
+  text-align: center;
+  font-weight: bold;
+  margin-top: 10%;
+}
+
+.option {
+  background-color: white;
+  border-radius: 10px;
+  align-items: center;
+  text-align: center;
+  margin-top: 15%;
+  padding: 0.3rem;
+}
+
+.logoShadow {
+  box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.5);
+}
+
+.containerVS {
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.footer {
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  left: 40%;
+  bottom: 5%;
+  width: 20%;
+  color: white;
+  text-align: center;
+}
```

Comment:
The button should be fixed, just leave it in the bottom of the other components.

Reply:
Ready